### PR TITLE
Use example dataset from PyGamma15 for testing

### PR DESCRIPTION
**Pull request by joleroi**
This PR sets up a yaml file parser that resolves environment variables. This way the example datastore created for the PyGamma15 workshope ($GAMMAPY_EXTRA/datasets/hess-crab4) can be used for testing.

### CODE REVIEW

**Comment by joleroi** [comment]
@cdeil Can you have a look if you have a better idea how to handle this?

File: gammapy/obs/data_store.py [hunk 1]
```diff
@@ -99,6 +99,25 @@ def from_name(cls, name):
         dm = DataManager()
         return dm[name]
 
+    @classmethod
```
Comment (joleroi) [inline gammapy/obs/data_store.py]: I need this for the spectrum analysis class. Usually on would pass a name, but for testing purposes the gammapy-extra repo has to be used

**Comment by cdeil** [comment]
Preferred:
https://docs.python.org/2/library/os.path.html#os.path.expandvars
```
from os.path import expandvars
from pathlib import Path
def make_path(path):
    return Path(expandvars(path))
```

**Comment by joleroi** [comment]
I merged this branch into the one I am currently working on

**Pull request closed without merging**